theme/src/components/skip-link.js

```diff
@@ -0,0 +1,38 @@
+import {Link} from '@primer/components'
+import styled from 'styled-components'
+import React from 'react'
+
+function SkipLinkBase({...rest}) {
```

Comment:
I don't think we need to use the spread operator here since we're not pulling off any properties.
```suggestion
function SkipLinkBase(props) {
```